homeassistant/components/toon/const.py

```diff
@@ -338,6 +340,51 @@
         ATTR_ICON: "mdi:solar-power",
         ATTR_DEFAULT_ENABLED: True,
     },
+    "water_average": {
+        ATTR_NAME: "Average Water Usage",
+        ATTR_SECTION: "water_usage",
+        ATTR_MEASUREMENT: "average",
+        ATTR_UNIT_OF_MEASUREMENT: VOLUME_LHOUR,
+        ATTR_DEVICE_CLASS: None,
+        ATTR_ICON: "mdi:water",
+        ATTR_DEFAULT_ENABLED: True,
```

Comment:
Having water usage available is rare, let's disable all those entities by default.